esp/src/src/nls/hu/hpcc.ts

```diff
@@ -487,6 +497,8 @@ export = {
     Lock: "Lezár",
     log_analysis_1: "log_elemzés_1*",
     Log: "Log",
+    LogAccess_GenericException: "ws_logaccess::GetLogAccessInfo, kivétel történt.",
```

Comment:
Please remove the ',' after '::GetLogAccessInfo'

Reply:
Change made, please review